src/aqua/installation-spell/src/aqua/config.aqua

```diff
@@ -56,6 +57,7 @@ data SpellDefinition:
 data WorkerDefinition:
     services: []Service
     spells: []SpellDefinition
+    total_memory_limit: u64
```

Comment:
it should be in `data Service`: it's a per-service limit

Reply:
damn my aim is really bad today.